addons/base_setup/views/res_config_views.xml

```diff
@@ -9,147 +9,240 @@
+                                    </div>
+                                    <div class="col-md-6 o_setting_box">
+                                        <div class="o_setting_left_pane">
+                                            <field name="module_google_spreadsheet" />
+                                        </div>
+                                        <div class="o_setting_right_pane">
+                                            <label for="module_google_spreadsheet"/>
+                                            <div class="text-muted">
+                                                Extract and analyze Odoo data from Google Spreadsheet
+                                            </div>
+                                            <div class="content-group" attrs="{'invisible': [('module_google_spreadsheet','=',False)]}" id="google_spreadsheet">
+                                                <div class="mt16 text-warning"><strong>Save</strong> this page to activate the feature.
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="col-md-6 o_setting_box">
+                                        <div class="input-group">
+                                            <div class="o_setting_left_pane">
+                                                <field name="module_auth_oauth" />
+                                            </div>
+                                            <div class="o_setting_right_pane">
+                                                <label string="OAuth Authentification" for="module_auth_oauth"/>
+                                                <div class="text-muted">
+                                                   Use external accounts to log in (Google, Facebook, etc.)
+                                                </div>
+                                                <div class="content-group" attrs="{'invisible': [('module_auth_oauth','=',False)]}" id="auth_oauth">
+                                                    <div class="mt16 text-warning"><strong>Save</strong> this page to activate the feature.
+                                                    </div>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="col-md-6 o_setting_box" id="google_oauth">
+                                    </div>
                                 </div>
                             </div>
+                            <div name="company_dependent_settings_message">
+                                <div>(*) This configuration is related to the company you're logged into.</div>
```

Comment:
is it required now? i don't think so. now we have set icon that will be visible on multi-company..